server/ws/server.go

```diff
@@ -520,3 +526,27 @@ func (ws *Server) BroadcastBlockChange(workspaceID string, block model.Block) {
 		}
 	}
 }
+
+// BroadcastConfigChange broadcasts update messages to clients.
+func (ws *Server) BroadcastConfigChange(clientConfig model.ClientConfig) {
+	message := UpdateClientConfig{
+		Action:       websocketActionUpdateConfig,
+		ClientConfig: clientConfig,
+	}
+
+	listeners := ws.listeners
+	ws.logger.Debug("listener(s)",
```

Comment:
nit: this message might be confusing without more context
```suggestion
	ws.logger.Debug("broadcasting config change to listener(s)",
```

Reply:
done